### PR TITLE
Multistepsignup

### PR DESCRIPTION
**Pull request by florindiconescu**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 <div class="container">
   <div class="back-section-space">
     <%= link_to "back".html_safe, :back, id:"back", class:"back-link" %>
-    <%= link_to 'edit', edit_user_path(@user) %>
+    <%= link_to 'edit', edit_user_registration_path %>
```
Comment (edgr) [inline app/views/users/show.html.erb]: So when editing, it goes back to the registration path?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -18,18 +18,21 @@ def index
   end
 
   def show
-    @user = User.find(params[:id])
-    @reviews = UserReview.where(receiver_id: @user.id)
-    unless @user.all_challenges.count == 0
-      @progressbar = ((@user.challenges_won.count / @user.all_challenges.count.to_f) * 100).to_i
+    @user = current_user
+    # @user = User.find(params[:id])
+    @reviews = UserReview.where(receiver: @user)
```
Comment (edgr) [inline app/controllers/users_controller.rb]: Are you sure this is working? In our schema, **receiver_id** is the column name.

File: db/seeds.rb [hunk 3]
```diff
@@ -35,10 +35,11 @@ def elo_points(challenger_score, challengee_score, challenge)
 puts "destroy all"
 
 User.destroy_all
+Club.destroy_all
 
 puts "creating users"
 
-names = ['Monica', 'Nam', 'Juan', 'Carlos', 'Christine', 'Phillip', 'Joel', 'Lukas', 'Jackson', 'Ricardo', 'Meder', 'Cristobal', 'Valentin', 'David', 'Claire', 'Folrin', 'Ashley']
+names = ['Monica', 'Nam', 'Juan', 'Carlos', 'Christine', 'Phillip', 'Joel', 'Lukas', 'Jackson', 'Ricardo', 'Meder', 'Cristobal', 'Valentin', 'David', 'Claire', 'Florin', 'Ashley']
```
Comment (edgr) [inline db/seeds.rb]: hahahaha :)

File: app/views/users/show.html.erb (hunk 1 above)
Comment (florindiconescu) [reply app/views/users/show.html.erb]: Yes, this is how it should be. Now you can think of some wireframes for edit user, so I can make that one as well?

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (florindiconescu) [reply app/controllers/users_controller.rb]: I guess so :) thru the power of active records
image

File: db/seeds.rb (hunk 3 above)
Comment (florindiconescu) [reply db/seeds.rb]: ;)

File: app/views/users/show.html.erb (hunk 1 above)
Comment (edgr) [reply app/views/users/show.html.erb]: Leo has built one especially for this a few days ago. It looks like a large form but that's ok for now. Can we change it back to that previous one?

**Comment by florindiconescu** [comment]
You'll have to bundle install and db:seed !

**Pull request merged**